.github/issue_template.md

```diff
@@ -0,0 +1,61 @@
+This template includes boilerplate for different kinds of issues. Select the
+one that is most suitable, delete first level header and the other templates.
+
+# Template 1: Bug
+
+Description of the problem
+
+## Affected platforms
+
+## Repro steps
+
+1. One
+2. Two
+3. Three
+
+## Expected behavior
+
+## Actual behavior
+
+# Template 2: Suggest new API
```

Comment:
maybe add a few more newlines before this header so the two templates are more visually separated.